src/dbus/NhekoDBusApi.h

```diff
@@ -58,6 +58,26 @@ class RoomInfoItem : public QObject
     int unreadNotifications_;
 };
 
+//! Get the nheko D-Bus API version.
+QString
+apiVersion();
+//! Get the nheko version.
+QString
+nhekoVersionString();
+//! Call this function to get a list of all joined rooms.
+QVector<RoomInfoItem>
+getRooms();
```

Comment:
If we are already breaking everything, maybe call this listRooms or rooms instead of getRooms? I haven't found any interface use "get*"